packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -702,3 +702,294 @@ describe('@api-conformity @conformity-batch-3 Ethereum execution apis tests', as
     synthesizeWsTestCases(TEST_CASES, updateParamIfNeeded);
   });
 });
+
+describe('@api-conformity @conformity-batch-4 Ethereum execution apis tests', async function () {
+  this.timeout(240 * 1000);
+
+  let existingCallerContractAddress: string;
+  let existingLogsContractAddress: string;
+  let fromBlockForLogs: string;
+
+  before(async () => {
+    const deployCallerContractTx = await signAndSendRawTransaction({
+      chainId: 0x12a,
+      to: null,
+      from: sendAccountAddress,
+      maxPriorityFeePerGas: gasPrice,
+      maxFeePerGas: gasPrice,
+      gasLimit: gasLimit,
+      type: 2,
+      data: CallerContract.bytecode,
+    });
+
+    const deployLogsContractTx = await signAndSendRawTransaction({
+      chainId: 0x12a,
+      to: null,
+      from: sendAccountAddress,
+      maxPriorityFeePerGas: gasPrice,
+      maxFeePerGas: gasPrice,
+      gasLimit: gasLimit,
+      type: 2,
+      data: LogsContract.bytecode,
+    });
+
+    existingCallerContractAddress = deployCallerContractTx.contractAddress;
+    existingLogsContractAddress = deployLogsContractTx.contractAddress;
+
+    const log0ContractCall = await signAndSendRawTransaction({
+      chainId: 0x12a,
+      to: existingLogsContractAddress,
+      from: sendAccountAddress,
+      maxPriorityFeePerGas: gasPrice,
+      maxFeePerGas: gasPrice,
+      gasLimit: gasLimit,
+      type: 2,
+      data: '0xd05285d4000000000000000000000000000000000000000000000000000000000000160c',
+    });
+
+    fromBlockForLogs = log0ContractCall.blockNumber;
+  });
+
+  const TEST_CASES = {
```

Comment:
wondering, if we should extract these in a separate file, since we may have to add more test cases in the future

Reply:
done.